src/Domain/Configuration.php

```diff
@@ -42,6 +42,12 @@ final class Configuration
         'disable-security-check',
     ];
 
+    private const ACCEPTED_FORMATS = [
+        'console',
+        'json',
+        'checkstyle'
```

Comment:
`github-action` format is missing. We have already a list of formatters in `src/Application/Console/Formatters/FormatResolver.php`, maybe we should use the same ?